external/fv3fit/fv3fit/reservoir/train.py

```diff
@@ -156,14 +137,22 @@ def train_reservoir_model(
     readout = combine_readouts(subdomain_readouts)
 
     model: Union[ReservoirComputingModel, HybridReservoirComputingModel]
+
+    # After training, the data used in inference does not have a time dimension
+    rank_divider_spatial_only = RankDivider(
```

Comment:
The `TimeSeriesRankDivider` is only useful for manipulating training data with a time dim, for prediction using a saved model we only work with spatial dimensions.